src/screens/MenuScreen.tsx

```diff
@@ -70,12 +97,13 @@ export default function MenuScreen() {
     return (
         <View style={styles.container}>
             <View style={styles.subContainer}>
-                <CardStack ref={swiper => { swiper = swiper }}>
-                    {randRecipes.map((rcp: Recipe) => {
-                        return <Card ><RecipeCard rcpImage={rcp.image} id={rcp.id} /></Card>
+                <CardStack style={styles.cardStack} ref={swiper => { swiper = swiper }} disableBottomSwipe disableTopSwipe>
+                    {randRecipes.map((rcp: Recipe, idx: number) => {
+                        return <Card onSwipedLeft={() => { onSwipedLeft(idx) }} onSwipedRight={() => { onSwipedRight(idx) }}><RecipeCard rcp={rcp} id={rcp.id} /></Card>
```

Comment:
Can we simply add key={rcp.id} to the <Card> component? I'm just seeing a Warning message about this.